tests/conftest.py

```diff
@@ -2,11 +2,31 @@
 
 from flask import Flask
 
+from flask_githubapp import GitHubApp
+
 
 @pytest.fixture
 def app():
     app = Flask('test_app')
     app.config['GITHUBAPP_ID'] = 1
     app.config['GITHUBAPP_KEY'] = 'key'
     app.config['GITHUBAPP_SECRET'] = 'secret'
-    return app
\ No newline at end of file
+    return app
+
+@pytest.fixture
+def github_app(app):
+    github_app = GitHubApp(app)
+
+    @github_app.on('issues')
+    def test_issue():
+        return 'issue event'
```

Comment:
I am not sure - but the github api docs will clear this out for sure - if events that support actions, like the `issues` event, will ever generate payloads _without_ actions. So this bit of test code might never be hit. However, I think it's good to test it!

Reply:
What this is trying to capture (and should be documented better) is that it's possible to create a function that responds to every action related to an event. So in this case using `@github_app.on('issues')` will make the function trigger regardless of which action is in the payload.

The only use case I've had for it so far is with check runs where was have a `check_run` handler that starts and updates a thread in a chat channel (so folks can be pulled in and given context quickly if things go awry) and additionally a `check_run.completed` handler that will DM the committer if checks fail.

We could've implemented all of that logic in the `check_run` handler, but it felt a little nicer to separate those as in one case we're trying to leave a bit of an audit trail in a place where it's easy to get multiple eyes quickly and in the other case we're trying to make sure someone knows the code they pushed had checks fail and can make a decision as to whether they need some help or if they happened to make a easy to fix error.

Reply:
Ah, reacting to events overall, and also to their actionbs, makes a lot of sense.